util/regexp-assemble/regexp-941130.data

```diff
@@ -0,0 +1,10 @@
+(?i)[\s\S]xlink:href\b
+(?i)[\s\S]xhtml\b
+(?i)[\s\S]xmlns\b
+(?i)[\s\S]!ENTITY.*?SYSTEM\b
+(?i)[\s\S]!ENTITY.*?PUBLIC\b
+(?i)[\s\S]data:text\/html\b
+(?i)[\s\S]formaction\b
+(?i)[\s\S]\@import\b
```

Comment:
Would you mind dropping the escape for / and @?

Reply:
Sure thing. We did not care enough to fix it initially. Good you noticed....